google-cloud-storage/acceptance/storage/bucket_test.rb

```diff
@@ -61,7 +61,7 @@
     _(one_off_bucket.website_404).must_equal "not_found.html"
     _(one_off_bucket.requester_pays).must_equal true
     # labels with symbols are not strings
-    _(one_off_bucket.labels).must_equal({ "foo" => "bar" })
+    _(one_off_bucket.labels).must_equal({ :foo => :bar })
```

Comment:
This line is - strictly speaking - not part of this PR's functionality. This test has been failing sporadically for the past few months (though the build has been reporting the error in line 42).

Reply:
This spec is still flaky & is failing. Reverting this change made it pass. I'll investigate it in a separate PR.